code/modules/mob/living/carbon/xenomorph/castes/gorger/abilities_gorger.dm

```diff
@@ -0,0 +1,188 @@
+// ***************************************
+// *********** Drain blood
+// ***************************************
+/datum/action/xeno_action/activable/drain
+	name = "Drain"
+	action_icon_state = "drain"
+	mechanics_text = "Hold a marine for 2s while draining 2% of their blood."
+	ability_name = "drain"
+	cooldown_timer = 15 SECONDS
+	plasma_cost = 50
+
+/datum/action/xeno_action/activable/drain/can_use_ability(atom/A, silent = FALSE, override_flags)
+	. = ..()
+	var/mob/living/carbon/xenomorph/gorger/X = owner
```

Comment:
put this right above the `if(X.blood_bank)` since you dont need it until then

Reply:
changed